apps/web/modules/ee/survey-follow-ups/components/FollowUpModal.tsx

```
@@ -0,0 +1,702 @@
+                                    return (
+                                      getLocalizedValue(ending.headline, selectedLanguageCode) || "Ending"
+                                    );
+                                  }
+
+                                  return ending.label || ending.url || "Ending";
+                                };
+
+                                return (
+                                  <Label
+                                    className="w-80 cursor-pointer rounded-md border border-slate-300 bg-slate-50 px-3 py-2 hover:bg-slate-100"
+                                    htmlFor={`ending-${ending.id}`}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)